{"name": "test_ct17", "status": "failed", "statusDetails": {"message": "AssertionError", "trace": "def test_ct17():\n        webdriver.get(url)\n        editar_atividade.caminho(False, True, False)\n>       editar_atividade.ct17_editar_atividade()\n\ntests\\test_editar_atividade.py:35: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <pages.page_editar_atividade.editarAtividade object at 0x0000016094319970>\n\n    def ct17_editar_atividade(self):\n        try:\n            self.find_element(self.btn_excluir).click()\n            sleep(1)\n            self.find_element(self.confirm_excluir).click()\n            sleep(1)\n            msg = self.espera_mensagem()\n            if msg is True:\n                if self.find_element(self.alert_texto).text == 'Atividade excluída com sucesso':\n                    print('\\n CT_17 sem erros: o atividade foi excluída com sucesso')\n                    assert True\n                else:\n                    print(\"\\n [!] CT_17 reportou erro: Não houve exclusão da atividade\")\n>                   assert False\nE                   AssertionError\n\npages\\page_editar_atividade.py:212: AssertionError"}, "start": 1606182444416, "stop": 1606182457068, "uuid": "c127f2d5-3ecc-4a11-bf40-e0bd5eeffc64", "historyId": "ab2f33b0c00079314da46d15615bce29", "testCaseId": "61a4abe89165e6558042ecce073cc74c", "fullName": "tests.test_editar_atividade#test_ct17", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_editar_atividade"}, {"name": "host", "value": "DESKTOP-OFT3SDL"}, {"name": "thread", "value": "14008-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_editar_atividade"}]}